app/src/features/settings/components/BillingTeam/components/RequestTeamAccess/components/Reminder/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { useFeatureValue } from "@growthbook/growthbook-react";
+import { getUserAuthDetails } from "store/selectors";
+import { Modal, Col } from "antd";
+import { getAvailableBillingTeams } from "store/features/billing/selectors";
+import { BillingTeamDetails } from "features/settings/components/BillingTeam/types";
+import { BillingTeamCard } from "../TeamCard";
+import { trackRequestBillingTeamAccessModalViewed } from "features/settings/analytics";
+import APP_CONSTANTS from "config/constants";
+import { openEmailClientWithDefaultEmailBody } from "utils/Misc";
+import "../../index.scss";
+
+export const RequestBillingTeamAccessReminder = () => {
+  const joinTeamReminder = useFeatureValue("join_team_reminder", null);
+  const user = useSelector(getUserAuthDetails);
+  const billingTeams = useSelector(getAvailableBillingTeams);
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isModalClosable, setIsModalClosable] = useState(true);
+
+  const emailSubject = "Request a new Billing Team";
+  const emailBody = `Hey Requestly Team\n\n We'd like to setup a new Billing Team. Could you please assist with the next step here?\n\nThanks\n${user?.details?.profile?.displayName}`;
+
+  useEffect(() => {
+    if (isModalVisible) {
+      trackRequestBillingTeamAccessModalViewed();
+    }
+  }, [isModalVisible]);
+
+  useEffect(() => {
+    if (joinTeamReminder) {
+      const lastReminderDate = localStorage.getItem(APP_CONSTANTS.STORAGE.LOCAL_STORAGE.LAST_REMINDER_DATE);
+      const currentDate = new Date();
+      const reminderStartDate = new Date(joinTeamReminder.startDate);
+      const persistenceDate = new Date(joinTeamReminder.persistenceDate);
+
+      if (currentDate >= persistenceDate) {
+        setIsModalClosable(false);
+        setIsModalVisible(true);
+        return;
+      }
+
+      if (lastReminderDate === currentDate.toISOString().slice(0, 10)) {
+        return;
+      }
+      if (!lastReminderDate || currentDate >= reminderStartDate) {
+        setIsModalVisible(true);
+      }
```

Comment:
shouldn't this be `and`? else it will behave unexpectedly in case of `lastReminderDate` not present in local storage AND `currentDate < reminderStartDate` (_if we decide to adjust `reminderStartDate` in the future_)

```suggestion
      if (!lastReminderDate && currentDate >= reminderStartDate) {
        setIsModalVisible(true);
      }
```